config/api/src/main/java/org/jboss/windup/config/metadata/LabelProviderRegistry.java

```diff
@@ -0,0 +1,59 @@
+package org.jboss.windup.config.metadata;
+
+import org.jboss.windup.config.GraphRewrite;
+import org.jboss.windup.config.LabelProvider;
+
+import java.util.*;
+
```

Comment:
Please add comments to describe the class .

Reply:
done